classes/webservice/WebserviceRequest.php

```diff
@@ -411,14 +411,14 @@ public function specificPriceCalculation($parameters)
             $id_currency = (int) (isset($value['currency']) ? $value['currency'] : Configuration::get('PS_CURRENCY_DEFAULT'));
             $id_group = (int) (isset($value['group']) ? $value['group'] : (int) Configuration::get('PS_CUSTOMER_GROUP'));
             $quantity = (int) (isset($value['quantity']) ? $value['quantity'] : 1);
-            $use_tax = (int) (isset($value['use_tax']) ? $value['use_tax'] : Configuration::get('PS_TAX'));
+            $use_tax = (bool) (isset($value['use_tax']) ? $value['use_tax'] : Configuration::get('PS_TAX'));
```

Comment:
Are we sure about this? Some legacy code expects integers instead of booleans

Reply:
Yes. Check `src/Core/Tax/TaxOptionsConfiguration.php` at l.79.